src/components/config_parser.cpp

```diff
@@ -292,4 +294,23 @@ bool config_parser::is_valid_name(const string& name) {
   return true;
 }
 
+string config_parser::parse_escaped_value(string&& value, const string& key) {
+  string cfg_value = value;
+  bool log = false;
+  auto backslash_pos = value.find_first_of('\\');
+  while (backslash_pos != string::npos) {
+    if (value[backslash_pos + 1] != '\\') {
+      backslash_pos = value.find_first_of('\\', backslash_pos + 1);
+      log = true;
+    } else {
+     value = value.replace(backslash_pos, 2, "\\");
+     backslash_pos = value.find_first_of('\\', backslash_pos + 1);
+    }
```

Comment:
You have some weird indentation here (2 spaces in the if branch and 1 in the else branch), please use 2 spaces.

Also setting the `backslash_pos` is the same in both branches, you can move it after the if-else block.